aspnetcore/tutorials/first-web-api.md

```diff
@@ -271,85 +275,77 @@ The following HTTP response is produced with the preceding call to `GetAll`:
 
-The `GetAll` method returns an [ActionResult\<T> type](xref:web-api/action-return-types#actionresultt-type). MVC automatically serializes the object to [JSON](https://www.json.org/) and writes the JSON into the body of the response message. The response code for this method is 200, assuming there are no unhandled exceptions. Unhandled exceptions are translated into 5xx errors.
+`ActionResult` return types can represent a wide range of HTTP status codes. For example, `GetById` can return two different status values:
 
-The `GetById` method returns `ActionResult<TodoItem>` The `ActionResult<T>` and `ActionResult` return types represents a wide range of return types. `GetById` has two different return types:
-
-* If no item matches the requested ID, the method returns a 404 error. Returning [NotFound](/dotnet/api/microsoft.aspnetcore.mvc.controllerbase.notfound) returns an HTTP 404 response.
+* If no item matches the requested ID, the method returns a 404 [NotFound](/dotnet/api/microsoft.aspnetcore.mvc.controllerbase.notfound) error code.
 * Otherwise, the method returns 200 with a JSON response body. Returning `item` results in an HTTP 200 response.
 
-## Work with Postman
+## Test the GetAll method
 
-This tutorial uses Postman to test the web api app.
+This tutorial uses Postman to test the web API.
 
 * Install [Postman](https://www.getpostman.com/apps)
 * Start the web app.
 * Start Postman.
 * Disable **SSL certificate verification**
   
   * From  **File > Settings** (**General* tab), disable **SSL certificate verification**.
-  * > [!WARNING]
+    > [!WARNING]
     > Re-enable SSL certificate verification after testing the controller.
 
 * Create a new request.
-* Set the HTTP method to **GET**.
-* Set the URI to `https://localhost:<port>/api/todo`. For example, `https://localhost:5001/api/todo`.
+  * Set the HTTP method to **GET**.
+  * Set the request URL to `https://localhost:<port>/api/todo`. For example, `https://localhost:5001/api/todo`.
 * Set **Two pane view** in Postman.
 * Select **Send**.
 
-![Postman with above request](first-web-api/_static/2pv.png)
-
-## Create CRUD methods
+![Postman with Get request](first-web-api/_static/2pv.png)
 
-In the following sections, `Create`, `Update`, and `Delete` methods are added to the controller.
-
-### Create
+## Add a Create method
 
 Add the following `Create` method:
 
 [!code-csharp[](first-web-api/samples/2.2/TodoApi/Controllers/TodoController.cs?name=snippet_Create)]
 
-The preceding code is an HTTP POST method, as indicated by the [[HttpPost]](/dotnet/api/microsoft.aspnetcore.mvc.httppostattribute) attribute. MVC gets the value of the to-do item from the body of the HTTP request.
+The preceding code is an HTTP POST method, as indicated by the [[HttpPost]](/dotnet/api/microsoft.aspnetcore.mvc.httppostattribute) attribute. The method gets the value of the to-do item from the body of the HTTP request.
```

Comment:
Where are you seeing an error?  It 404s on staging because the relative link looks for a review.docs... link.  Remove the "review." and you get the page you'll see in production.